docs/source/task_guides/image_classification_lora.mdx

```diff
@@ -0,0 +1,431 @@
+
+We also specify `modules_to_save`. After wrapping the base model with `get_peft_model()` along with the `config`, we get
+a new model where only the LoRA parameters are trainable (so-called "update matrices") while the pre-trained parameters
+are kept frozen. However, we want the classifier parameters to be trained too when fine-tuning the base model on our
+custom dataset. To ensure that the classifier parameters are also trained, we specify `modules_to_save`. This also
+ensures that these modules are serialized alongside the LoRA trainable parameters when using utilities like `save_pretrained()`
+and `push_to_hub()`.
+
+Here's what the other parameters mean:
+
+`r`: The dimension used by the LoRA update matrices.
+`alpha`: Scaling factor.
+`bias`: Specifies if the `bias` parameters should be trained. `None` denotes none of the `bias` parameters will be trained.
+
+`r` and `alpha` together control the total number of final trainable parameters when using LoRA, giving you the flexibility
+to balance a trade-off between end performance and compute efficiency.
+
+By looking at the number of trainable parameters, you can see how many parameters we're actually training. Since the goal is
+to achieve parameter-efficient fine-tuning, you should expect to see fewer trainable parameters in the `lora_model`
+in comparison to the original model, which is indeed the case here.
+
+## Define training arguments
+
+For model fine-tuning, use [🤗 Trainer](https://huggingface.co/docs/transformers/main_classes/trainer). It accepts
+several arguments which you can wrap using `TrainingArguments`.
+
+```python
+from transformers import TrainingArguments, Trainer
+
+
+model_name = model_checkpoint.split("/")[-1]
+batch_size = 128
+
+args = TrainingArguments(
+    f"{model_name}-finetuned-lora-food101",
+    remove_unused_columns=False,
+    evaluation_strategy="epoch",
+    save_strategy="epoch",
+    learning_rate=5e-3,
+    per_device_train_batch_size=batch_size,
+    gradient_accumulation_steps=4,
+    per_device_eval_batch_size=batch_size,
+    fp16=True,
+    num_train_epochs=5,
+    logging_steps=10,
+    load_best_model_at_end=True,
+    metric_for_best_model="accuracy",
+    push_to_hub=True,
+    label_names=["labels"],
+)
+```
+
+Compared to fine-tuning the original model, you can use a larger batch size since there is only a handful of parameters to train.
+You can also set a larger learning rate than the normal (1e-5 for example).
+
+This is a byproduct of the fact that the training affects only a small number of parameters. This can
+potentially also reduce the need to conduct expensive hyperparameter tuning experiments.
+
+## Prepare evaluation metric
+
+```python
+import numpy as np
+import evaluate
+
+metric = evaluate.load("accuracy")
+
+
+# the compute_metrics function takes a Named Tuple as input:
+# predictions, which are the logits of the model as Numpy arrays,
+# and label_ids, which are the ground-truth labels as Numpy arrays.
```

Comment:
I think it'd be easier to read if this was regular text and not included in the code example